apps/graphql/app.js

```diff
@@ -0,0 +1,39 @@
+const express = require("express");
+const morgan = require("morgan");
+const bodyParser = require("body-parser");
+
+const config = require("./config");
+const { server } = require("./src/graphql");
+const secure = require("./src/secure");
+const development = require("./src/development");
+
+const app = secure(express());
+if (process.env.NODE_ENV !== "production") {
+  development(app);
+}
+
```

Comment:
Maybe it was a better idea to separate the Autofill and Netlify PR's. Now the whole context changes.

Reply:
you are correct. We'll do this differently in the future.